package.json

```diff
@@ -8,6 +8,7 @@
     "test": "tests"
   },
   "scripts": {
+    "build:json": "cd services/app-api && yarn run build:json",
```

Comment:
Do we have a plan for how and where to use this command? I think I still prefer [doing this generation at time of report create](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/pull/1099#pullrequestreview-3350516763). What do others think?